oauth2client/devshell.py

```diff
@@ -117,6 +124,10 @@ def __init__(self, user_agent=None):
     def _refresh(self, http_request):
         self.devshell_response = _SendRecv()
         self.access_token = self.devshell_response.access_token
+        expires_in = self.devshell_response.expires_in
+        if expires_in is not None:
```

Comment:
@nathanielmanistaatgoogle The fact that `self.access_token` means we have a brand new token state.

@wilford I'd argue that `self.token_expiry = None` should occur in the case that `expires_in is None` (the other branch of this `if` statement`)